docs/src/main/asciidoc/transaction.adoc

```diff
@@ -367,10 +367,13 @@ which is a CDI bean and can be ``@Inject``ed.
 
 In cloud environments where persistent storage is not available, such as when application containers are unable to use persistent volumes, you can configure the transaction management to store transaction logs in a database by using a JDBC datasource.
 
-IMPORTANT: While there are several benefits to using a database to store transaction logs, you might notice a reduction in performance compared with using the file system to store the logs.
+NOTE: This configuration is only relevant for _Jakarta Transactions_ transactions.
 
-Quarkus allows the following JDBC-specific configuration of the object store included in `quarkus.transaction-manager.object-store.<property>` properties, where <property> can be:
+However, in cloud-native apps, it is important to assess JTA transactions after careful evaluation.
+The Narayana framework, which manages these transactions, requires stable storage, a unique reusable node identifier, and a steady IP address to work correctly.
```

Comment:
I'd stick to the name of the extension rather than using terms like `Narayana framework`.

Reply:
+1